### ISSUE TITLE
Base docker image broken 

### ISSUE THREAD

**Issue by liashukvladyslav** [IgorKha/Grafana-Mikrotik#34]
snmp_exporter of v0.20.0, embedded in this repo's dockerfile, won't work with proposed config
 
```
level=info ts=2023-12-21T17:06:38.467Z caller=main.go:152 msg="Starting snmp_exporter" version="(version=0.20.0, branch=HEAD, revision=c33572b6c8c8e43a479fde0f9fa8ac86e15598bc)"
level=info ts=2023-12-21T17:06:38.467Z caller=main.go:153 build_context="(go=go1.15.8, user=root@eebd39e6960e, date=20210212-11:37:48)"
level=error ts=2023-12-21T17:06:38.482Z caller=main.go:159 msg="Error parsing config file" err="yaml: unmarshal errors:\n  line 3: field public_v2 not found in type config.plain\n  line 10: field mikrotik not found in type config.plain"
```

**Comment by IgorKha** [comment]
Hello! @liashukvladyslav, please use latest version
This is a known issue #31

**Comment by liashukvladyslav** [comment]
Actually, this is the latest version of `snmp.yml`

```
$ md5sum snmp.yml
123b9c7138fb37505285ea5d5eb42425  snmp.yml
$ head snmp.yml
    # WARNING: This file was auto-generated using snmp_exporter generator, manual changes will be lost.
auths:
    public_v2:
        community: public
        security_level: noAuthNoPriv
        auth_protocol: MD5
        priv_protocol: DES
        version: 2
modules:
    mikrotik:
$ git log | head
commit da35426cfddf3ce0a664fe6adc79d9d2e6766a4e
Author: IgorKha <2268209+IgorKha@users.noreply.github.com>
Date:   Thu Dec 21 13:50:05 2023 +0500

    Update README.md
```

**Comment by IgorKha** [comment]
please read issue #31 
TL;DR; you need use latest snmp_exporter
[detail](https://github.com/prometheus/snmp_exporter/blob/main/auth-split-migration.md)
